web/src/app/sidebar/interested-courses/component.scss

```diff
@@ -1 +1,14 @@
 @import "../../../_vars.scss";
+
+.schedule-status {
+  margin: 0;
+  margin-top: 3px;
+  margin-left: 5px;
+  font-size: 100%;
+  font-weight: 600;
+  min-width: 500px;
```

Comment:
I've never used @media before. Would it be used similarly to:
```
.schedule-status {
   ...code...
}
@media only screen and (min-widtth: 375px) {
   margin-left: 3px;
   font-size: 80%;
   min-width: 200px;
}
```

or at least something sort of like those values. Also, how would I go about testing to see if this would work on phones on just my computer?

Reply:
Hi yes that is pretty close. For an exact example, see https://github.com/YACS-RCOS/yacs/blob/feature/ui-overhaul/web/src/app/listing/component.scss#L24 and https://github.com/YACS-RCOS/yacs/blob/feature/ui-overhaul/web/src/app/listing/component.scss#L10

You'll need one `@media (min-width: 576px)` (for non-mobile) and one `@media (max-width: 575.98px)` for mobile.

Reply:
Updated.